lib/components/viewers/route-details.tsx

```diff
@@ -231,11 +248,22 @@ class RouteDetails extends Component<Props> {
   }
 }
 
+const mapStateToProps = (state: any) => {
+  const { sortRoutePatternsByVehicleCount } = state.otp.config.routeViewer
```

Comment:
```suggestion
  const { sortRoutePatternsByVehicleCount } = state.otp.config?.routeViewer
```
route viewer is not guaranteed to be present!